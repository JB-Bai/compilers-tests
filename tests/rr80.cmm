float;
struct{
	float GWQqatmn8 = 889.030E8;
	float WEDTaf[07140621][01452371][0xaeb][663], nmC5kK =(732.30e+7) - 479, BEBW7, jPTjSd;
}
;
float qRFp5FVtl(int RkH869Zl, int UkdONRr[0543][0x92]){
	int iDGfX2kSRK[0x216], EpTu426[0xCd][463171], zeYVTNQHMWf[452][415001] = 40.6, nWCFACFP5n, k5zf0R[0xEb] = 033.005e+9;
	0919.44 = - 1352.428e6;
	0760102 / 1.142e4;
	if(! 045203) if(rDV6) KZ5_V4_L5F7();
	if(VleOxiYTU ||(0440 || LsTHKmgWx(0513, 60.347) / OjVCtXSX - 2.20e+1) * 25.892e2.AIMH6Zo) - 93.681e5 && 03271 / 57672 == 6.36e-0;
	vw4zSlF;
}

