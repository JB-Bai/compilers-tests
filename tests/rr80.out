Program (1)
  ExtDefList (1)
    ExtDef (1)
      Specifier (1)
        TYPE: float
      SEMI
    ExtDefList (2)
      ExtDef (2)
        Specifier (2)
          StructSpecifier (2)
            STRUCT
            LC
            DefList (3)
              Def (3)
                Specifier (3)
                  TYPE: float
                DecList (3)
                  Dec (3)
                    VarDec (3)
                      ID: GWQqatmn8
                    ASSIGNOP
                    Exp (3)
                      FLOAT: 88903000064.000000
                SEMI
              DefList (4)
                Def (4)
                  Specifier (4)
                    TYPE: float
                  DecList (4)
                    Dec (4)
                      VarDec (4)
                        VarDec (4)
                          VarDec (4)
                            VarDec (4)
                              VarDec (4)
                                ID: WEDTaf
                              LB
                              INT: 1884561
                              RB
                            LB
                            INT: 414969
                            RB
                          LB
                          INT: 2795
                          RB
                        LB
                        INT: 663
                        RB
                    COMMA
                    DecList (4)
                      Dec (4)
                        VarDec (4)
                          ID: nmC5kK
                        ASSIGNOP
                        Exp (4)
                          Exp (4)
                            LP
                            Exp (4)
                              FLOAT: 7322999808.000000
                            RP
                          MINUS
                          Exp (4)
                            INT: 479
                      COMMA
                      DecList (4)
                        Dec (4)
                          VarDec (4)
                            ID: BEBW7
                        COMMA
                        DecList (4)
                          Dec (4)
                            VarDec (4)
                              ID: jPTjSd
                  SEMI
            RC
        SEMI
      ExtDefList (7)
        ExtDef (7)
          Specifier (7)
            TYPE: float
          FunDec (7)
            ID: qRFp5FVtl
            LP
            VarList (7)
              ParamDec (7)
                Specifier (7)
                  TYPE: int
                VarDec (7)
                  ID: RkH869Zl
              COMMA
              VarList (7)
                ParamDec (7)
                  Specifier (7)
                    TYPE: int
                  VarDec (7)
                    VarDec (7)
                      VarDec (7)
                        ID: UkdONRr
                      LB
                      INT: 355
                      RB
                    LB
                    INT: 146
                    RB
            RP
          CompSt (7)
            LC
            DefList (8)
              Def (8)
                Specifier (8)
                  TYPE: int
                DecList (8)
                  Dec (8)
                    VarDec (8)
                      VarDec (8)
                        ID: iDGfX2kSRK
                      LB
                      INT: 534
                      RB
                  COMMA
                  DecList (8)
                    Dec (8)
                      VarDec (8)
                        VarDec (8)
                          VarDec (8)
                            ID: EpTu426
                          LB
                          INT: 205
                          RB
                        LB
                        INT: 463171
                        RB
                    COMMA
                    DecList (8)
                      Dec (8)
                        VarDec (8)
                          VarDec (8)
                            VarDec (8)
                              ID: zeYVTNQHMWf
                            LB
                            INT: 452
                            RB
                          LB
                          INT: 415001
                          RB
                        ASSIGNOP
                        Exp (8)
                          FLOAT: 40.599998
                      COMMA
                      DecList (8)
                        Dec (8)
                          VarDec (8)
                            ID: nWCFACFP5n
                        COMMA
                        DecList (8)
                          Dec (8)
                            VarDec (8)
                              VarDec (8)
                                ID: k5zf0R
                              LB
                              INT: 235
                              RB
                            ASSIGNOP
                            Exp (8)
                              FLOAT: 33005000704.000000
                SEMI
            StmtList (9)
              Stmt (9)
                Exp (9)
                  Exp (9)
                    FLOAT: 919.440002
                  ASSIGNOP
                  Exp (9)
                    MINUS
                    Exp (9)
                      FLOAT: 1352428032.000000
                SEMI
              StmtList (10)
                Stmt (10)
                  Exp (10)
                    Exp (10)
                      INT: 254018
                    DIV
                    Exp (10)
                      FLOAT: 11420.000000
                  SEMI
                StmtList (11)
                  Stmt (11)
                    IF
                    LP
                    Exp (11)
                      NOT
                      Exp (11)
                        INT: 19075
                    RP
                    Stmt (11)
                      IF
                      LP
                      Exp (11)
                        ID: rDV6
                      RP
                      Stmt (11)
                        Exp (11)
                          ID: KZ5_V4_L5F7
                          LP
                          RP
                        SEMI
                  StmtList (12)
                    Stmt (12)
                      IF
                      LP
                      Exp (12)
                        Exp (12)
                          ID: VleOxiYTU
                        OR
                        Exp (12)
                          Exp (12)
                            LP
                            Exp (12)
                              Exp (12)
                                INT: 288
                              OR
                              Exp (12)
                                Exp (12)
                                  Exp (12)
                                    ID: LsTHKmgWx
                                    LP
                                    Args (12)
                                      Exp (12)
                                        INT: 331
                                      COMMA
                                      Args (12)
                                        Exp (12)
                                          FLOAT: 60.347000
                                    RP
                                  DIV
                                  Exp (12)
                                    ID: OjVCtXSX
                                MINUS
                                Exp (12)
                                  FLOAT: 22.000000
                            RP
                          STAR
                          Exp (12)
                            Exp (12)
                              FLOAT: 2589.199951
                            DOT
                            ID: AIMH6Zo
                      RP
                      Stmt (12)
                        Exp (12)
                          Exp (12)
                            MINUS
                            Exp (12)
                              FLOAT: 9368100.000000
                          AND
                          Exp (12)
                            Exp (12)
                              Exp (12)
                                INT: 1721
                              DIV
                              Exp (12)
                                INT: 57672
                            RELOP
                            Exp (12)
                              FLOAT: 6.360000
                        SEMI
                    StmtList (13)
                      Stmt (13)
                        Exp (13)
                          ID: vw4zSlF
                        SEMI
            RC
